aggregator/options.go

```diff
@@ -796,7 +797,11 @@ func (o *options) computeDefaultSuffixes() {
 		default:
 			q, ok := aggType.Quantile()
 			if ok {
-				o.defaultSuffixes[aggType.ID()] = o.timerQuantileSuffixFn(q)
+				b, err := o.timerQuantileSuffixFn(q)
```

Comment:
Perhaps it'll be better to still have the function return a `[]byte` and panic inside the function on error instead?